.gitattributes

```diff
@@ -12,6 +12,5 @@ hugegraph-store/hg-store-dist/src/assembly/static/bin/libjemalloc_aarch64.so exp
 .github/ export-ignore
 .idea/ export-ignore
 install-dist/scripts/ export-ignore
-install-dist/release-docs/ export-ignore
```

Comment:
because of https://github.com/apache/incubator-hugegraph/blob/master/install-dist/pom.xml#L56